From 08abf944b2bff27b6dd325dd15b4861cebef113a Mon Sep 17 00:00:00 2001
From: Junling Zheng <zhengjunling@huawei.com>
Date: Wed, 8 Apr 2015 03:26:57 +0000
Subject: [PATCH] Add -P prompt into Usage info

Currently, Usage info doesn't include -P prompt, add it:

root@arma9el:~# file
Usage: file [-bcEhikLlNnprsvz0] [--apple] [--mime-encoding] [--mime-type]
            [-e testname] [-F separator] [-f namefile] [-m magicfiles]
            [-P parameter=limits] file ...
       file -C [-m magicfiles]
       file [--help]

Upstream-Status: Submitted

Signed-off-by: Junling Zheng <zhengjunling@huawei.com>
---
 src/file.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/file.c b/src/file.c
index 2b15e21..e5a957b 100644
--- a/src/file.c
+++ b/src/file.c
@@ -76,8 +76,8 @@ int getopt_long(int argc, char * const *argv, const char *optstring, const struc
 # define USAGE  \
     "Usage: %s [" FILE_FLAGS \
 	"] [--apple] [--extension] [--mime-encoding] [--mime-type]\n" \
-    "            [-e testname] [-F separator] [-f namefile] [-m magicfiles] " \
-    "file ...\n" \
+    "            [-e testname] [-F separator] [-f namefile] [-m magicfiles]\n " \
+    "            [-P parameter=limits] file ...\n" \
     "       %s -C [-m magicfiles]\n" \
     "       %s [--help]\n"
 
-- 
1.8.3.4

